lib/zwave/system/capabilities/volume_set/SOUND_SWITCH.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
+  get: 'SOUND_SWITCH_CONFIGURATION_GET',
+  set: 'SOUND_SWITCH_CONFIGURATION_SET',
+  getOpts: {
+    getOnStart: true,
+  },
+  setParserV1(value) {
+    return {
+      Volume: value * 100,
+      'Default Tone Identifier': 1
+    }
+  },
+  report: 'SOUND_SWITCH_CONFIGURATION_REPORT',
+  reportParserV1: report => {
+    if (report && report.hasOwnProperty('Volume')) {
```

Comment:
In the unexpected case that the report contains a `Volume` property that is `undefined` the report parser will return `NaN`. Best to make these parsers as defensive as possible by adding a type check.